integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_files_excluded.py

```diff
@@ -0,0 +1,172 @@
+                        "name": collection_name,
+                        "version": collection_version
+                        }
+                },
+                "cumulus_meta": {
+                    "execution_name": execution_id
+                },
+                "task_config": {
+                    "excludedFileExtensions": "{$.meta.collection.meta.orca.excludedFileExtensions}",
+                    "s3MultipartChunksizeMb": "{$.meta.collection.meta.s3MultipartChunksizeMb}",
+                    "providerId": "{$.meta.provider.id}",
+                    "providerName": "{$.meta.provider.name}",
+                    "executionId": "{$.cumulus_meta.execution_name}",
+                    "collectionShortname": "{$.meta.collection.name}",
+                    "collectionVersion": "{$.meta.collection.version}",
+                    "defaultBucketOverride": "{$.meta.collection.meta.orca.defaultBucketOverride}",
+                    "defaultStorageClassOverride": "{$.meta.collection.meta.orca.defaultStorageClassOverride}"
+                },
+                "exception": "None"
+                }
+
+            execution_info = boto3.client("stepfunctions").start_execution(
+                stateMachineArn=helpers.orca_copy_to_archive_step_function_arn,
+                input=json.dumps(copy_to_archive_input, indent=4),
+            )
+
+            step_function_results = helpers.get_state_machine_execution_results(
+                execution_info["executionArn"]
+            )
+
+            self.assertEqual(
+                "SUCCEEDED",
+                step_function_results["status"],
+                "Error occurred while starting step function.",
+            )
+            self.assertEqual(
+                expected_output,
+                json.loads(step_function_results["output"]),
+                "Expected step function output not returned.",
+            )
+            catalog_output = helpers.post_to_api(
+                my_session,
+                helpers.api_url + "/catalog/reconcile/",
+                data=json.dumps(
+                    {
+                        "pageIndex": 0,
+                        "granuleId": [granule_id],
+                        "endTimestamp": int((time.time() + 5) * 1000),
+                    }
+                ),
+                headers={"Host": helpers.aws_api_name},
+            )
+            self.assertEqual(
+                200, catalog_output.status_code, "Error occurred while contacting API."
+            )
+            self.assertEqual(
+                {
+                    "anotherPage": False,
+                    "granules": [
+                        {   "id": granule_id,
+                            "collectionId": collection_name + "___" + collection_version,
+                            "createdAt": createdAt_time,
+                            "executionId": execution_id,
+                            "providerId": provider_id,
+                            "files": [],
```

Comment:
coube possibly be due to files being ingested already

Reply:
In this case, files: [] is likely valid, since all files should be excluded.

Reply:
yup